content/src/debug.js

```diff
@@ -175,6 +219,125 @@ function detectStuckProgram() {
 }
 
 //////////////////////////////////////////////////////////////////////
+// VARIABLE & FUNCTION CALL TRACKING
+//////////////////////////////////////////////////////////////////////
+
+var untrackedVariables = [
```

Comment:
Is there a way for us to collect this list automatically instead of listing them here?  For example, we have a short script "showturtle.js" which is run at the end of turtlebits.js.  Maybe it could iterate through the window object and feed predefined global symbols into the IDE?